frappe/model/mapper.py

```diff
@@ -152,6 +152,9 @@ def get_mapped_doc(
 							True if target_doc.get(target_parentfield) else False
 						)
 
+					if table_map.get("ignore"):
```

Comment:
We can just remove it in post-process right? Why add another hook for it then?

Reply:
Also `ignore_fields` already exists, it should doing the job of not mapping a child table too.

Reply:
There's also `condition`: 
 

`lambda child: False`